app/src/main/java/org/smartregister/addo/application/AddoSyncConfiguration.java

```diff
@@ -31,9 +31,11 @@ public String getSyncFilterValue() {
 
         List<String> locationIds = LocationHelper.getInstance().locationsFromHierarchy(true, null);
         if (!org.smartregister.util.Utils.isEmptyCollection(locationIds)) {
-            int index = locationIds.indexOf(userLocationId);
-            List<String> subLocationIds = locationIds.subList(index, locationIds.size());
-            return StringUtils.join(subLocationIds, ",");
+/*            int index = locationIds.indexOf(userLocationId);
+            List<String> subLocationIds = locationIds.subList(index, locationIds.size());*/
+            locationIds.add("05c5bb5e-0417-442f-b3be-f702e22ffb3f"); // This location id is for Tumbi where most test users
```

Comment:
Am guessing hardcoding this won't work going live?

Reply:
@Andati - yes this hard coded locationId is for the Tumbi location that has most of users. I am putting it here so that the app will have more user data for testing. I will remove it and add it only on development branch.